File: dotnet/Samples/Dictionaries/DictionaryService.cs
using LanguageWeaver.Sdk.Common;
using LanguageWeaver.Sdk.Configurations;
using LanguageWeaver.Sdk.Dictionaries.Common.Result;

namespace LanguageWeaver.Sdk.Samples.Dictionaries;

public static class DictionaryService
{
    public static readonly int PAGE_NUMBER = 1;
    public static readonly int PAGE_SIZE = 100;

    public static void Main()
    {
        using var lwClient = new SdkFactory().GetLanguageWeaverClient(new ClientConfiguration());
        DictionariesResult dictionariesResult = lwClient.GetDictionaries(PAGE_NUMBER, PAGE_SIZE);

        dictionariesResult.Dictionaries
            .ForEach(dictionary => Console.WriteLine(dictionary.Id));
    }
}

File: javascript/backend/samples/dictionaries/DictionaryService.js
const SdkFactory = require("@language-weaver/lw-sdk-js").default;
const {ClientConfiguration} = require("@language-weaver/lw-sdk-js");

const getDictionaries = async () => {
    try {
        const lwClient = await SdkFactory.getLanguageWeaverClient(new ClientConfiguration());
        const dictionariesResult = await lwClient.getDictionaries(1);
        console.log(dictionariesResult.dictionaries);
    } catch (e) {
        console.log(e);
    }
}

getDictionaries();